requirements.txt

```diff
@@ -1,5 +1,6 @@
-plotly<6.1
-# not imported explicitly, but used by plotly to render PNGs
+# Kaleido is not imported directly, but it is needed for Plotly to render PNGs.
 kaleido<0.3
-sqlalchemy<2.1
+numpy<2
+pandas<2.3
```

Comment:
I wonder if notebooks still can trip, when not pinning dependencies over there as well.
The patch here effectively just satisfies the test suite, because notebooks do not use `pip install` to install dependencies from this `requirements.txt` file, do they?